modules/dag-l0/src/main/scala/org/tessellation/dag/l0/infrastructure/snapshot/GlobalSnapshotConsensus.scala

```diff
@@ -92,7 +95,13 @@ object GlobalSnapshotConsensus {
         snapshotAcceptanceManager,
         collateral,
         rewards,
-        GlobalSnapshotEventCutter.make[F](snapshotConfig.consensus.eventCutter.maxBinarySizeBytes)
+        GlobalSnapshotEventCutter.make[F](
+          snapshotConfig.consensus.eventCutter.maxBinarySizeBytes,
+          // TODO pull fee calculator config after Maciej's PR merged
```

Comment:
isn't it already merged? 🤔

Reply:
@marcinwadon Yes -- thank you for catching that. Updated.